### ISSUE TITLE
Protect route `/listen/:channel_id` by session

### ISSUE THREAD

**Issue by hbredin** [marc-j/camomile-server#8]
As discussed in #4,  route `/listen/:channel` is not protected by session -- while it probably should be.

**Comment by marc-j** [comment]
Yes, because :channel is a hash generated by secured route ans it's not possible to retrieve this if not authenticated

**Comment by hbredin** [comment]
Yes, I know -- I just opened this issue for reference... (in case we want to improve this particular weakness later...)
